### PR TITLE
Fix source URL and add alternate checksum for Hypre 2.14.0

### PR DESCRIPTION
**Pull request by tardigradus**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node371.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/848a8a82c5c09812cdb13b081bf3acdc for a full test report.

**Comment by lexming** [comment]
The new tarball just differs in the provided documentation

> $ rsync -rlcvn --delete hypre-2.14.0/ hypre-2.14.0.orig/
sending incremental file list
deleting docs/HYPRE_usr_manual.pdf
deleting docs/HYPRE_ref_manual.pdf
deleting docs/
deleting src/examples/README_files/vis.c.html
deleting src/examples/README_files/syspfmg.html
deleting src/examples/README_files/struct.html
deleting src/examples/README_files/sstruct.html
deleting src/examples/README_files/smg.html
deleting src/examples/README_files/pfmg.html
deleting src/examples/README_files/pcg.html
deleting src/examples/README_files/pcg-split-smg.html
deleting src/examples/README_files/pcg-smg.html
deleting src/examples/README_files/pcg-pfmg.html
deleting src/examples/README_files/pcg-parasails.html
deleting src/examples/README_files/pcg-euclid.html
deleting src/examples/README_files/pcg-ams.html
deleting src/examples/README_files/pcg-amg.html
deleting src/examples/README_files/lobpcg-amg.html
deleting src/examples/README_files/ij.html
deleting src/examples/README_files/hypre_wiw.gif
deleting src/examples/README_files/gmres-syspfmg.html
deleting src/examples/README_files/gmres-smg.html
deleting src/examples/README_files/gmres-pfmg.html
deleting src/examples/README_files/gmres-parasails.html
deleting src/examples/README_files/gmres-euclid.html
deleting src/examples/README_files/gmres-amg.html
deleting src/examples/README_files/fortran.html
deleting src/examples/README_files/flexgmres-amg.html
deleting src/examples/README_files/fei.html
deleting src/examples/README_files/fe.html
deleting src/examples/README_files/ex9.html
deleting src/examples/README_files/ex9.c.html
deleting src/examples/README_files/ex8.html
deleting src/examples/README_files/ex8.c.html
deleting src/examples/README_files/ex7.html
deleting src/examples/README_files/ex7.c.html
deleting src/examples/README_files/ex6.html
deleting src/examples/README_files/ex6.c.html
deleting src/examples/README_files/ex5f.html
deleting src/examples/README_files/ex5f.f.html
deleting src/examples/README_files/ex5big.html
deleting src/examples/README_files/ex5big.c.html
deleting src/examples/README_files/ex5.html
deleting src/examples/README_files/ex5.c.html
deleting src/examples/README_files/ex4.html
deleting src/examples/README_files/ex4.c.html
deleting src/examples/README_files/ex3.html
deleting src/examples/README_files/ex3.c.html
deleting src/examples/README_files/ex2.html
deleting src/examples/README_files/ex2.c.html
deleting src/examples/README_files/ex18comp.c.html
deleting src/examples/README_files/ex18.c.html
deleting src/examples/README_files/ex17.c.html
deleting src/examples/README_files/ex16.html
deleting src/examples/README_files/ex16.c.html
deleting src/examples/README_files/ex15big.html
deleting src/examples/README_files/ex15big.c.html
deleting src/examples/README_files/ex15.html
deleting src/examples/README_files/ex15.c.html
deleting src/examples/README_files/ex14.html
deleting src/examples/README_files/ex14.c.html
deleting src/examples/README_files/ex13.html
deleting src/examples/README_files/ex13.c.html
deleting src/examples/README_files/ex12f.html
deleting src/examples/README_files/ex12f.f.html
deleting src/examples/README_files/ex12.html
deleting src/examples/README_files/ex12.c.html
deleting src/examples/README_files/ex11.html
deleting src/examples/README_files/ex11.c.html
deleting src/examples/README_files/ex10.html
deleting src/examples/README_files/ex10.cxx.html
deleting src/examples/README_files/ex1.html
deleting src/examples/README_files/ex1.c.html
deleting src/examples/README_files/c.html
deleting src/examples/README_files/c++.html
deleting src/examples/README_files/amg.html
deleting src/examples/README_files/9pt.html
deleting src/examples/README_files/5pt.html
deleting src/examples/README_files/3d-maxwell.html
deleting src/examples/README_files/2d-laplace.html
deleting src/examples/README_files/2d-laplace-eig.html
deleting src/examples/README_files/2d-con-reac-diff.html
deleting src/examples/README_files/2d-bi.html
deleting src/examples/README_files/
deleting src/examples/README.html

File: easybuild/easyconfigs/h/Hypre/Hypre-2.14.0-foss-2018a.eb [hunk 1]
```diff
@@ -9,9 +9,9 @@ description = """Hypre is a library for solving large, sparse linear systems of
 toolchain = {'name': 'foss', 'version': '2018a'}
 toolchainopts = {'pic': True}
 
-source_urls = ['https://github.com/LLNL/hypre/archive/']
+source_urls = ['https://github.com/hypre-space/hypre/archive/']
 sources = ['v%(version)s.tar.gz']
-checksums = ['10cfcd555618137c194958f84f44724fece45b58c59002d1195fed354e2ca16c']
+checksums = ['705a0c67c68936bb011c50e7aa8d7d8b9693665a9709b584275ec3782e03ee8c']
```
Comment (lexming) [inline easybuild/easyconfigs/h/Hypre/Hypre-2.14.0-foss-2018a.eb:14]: We should not remove the old checksum as there might be users that already have the old `v2.14.0.tar.gz` tarball and still might want to install this easyconfig of Hypre
```suggestion
checksums = [('10cfcd555618137c194958f84f44724fece45b58c59002d1195fed354e2ca16c',
              '705a0c67c68936bb011c50e7aa8d7d8b9693665a9709b584275ec3782e03ee8c')]
```

**Review by lexming — CHANGES_REQUESTED** [review]
The new tarball is fine. I just have a suggestion regarding the checksums. Please see below

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3100.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/611d84e4e77c8252be7336f8df754035 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @tardigradus!

**Comment by boegel** [comment]
Going in, thanks @tardigradus!

**Pull request merged**